apis/beacon/states/validator_identities.yaml

```diff
@@ -0,0 +1,71 @@
+post:
+  operationId: "postStateValidatorIdentities"
+  summary: "Get validator identities from state"
+  description: |
+    Returns filterable list of validators identities.
+
+    Identities will be returned for all indices or public keys that match known validators. If an index or public key does not
+    match any known validator, no identity will be returned but this will not cause an error. There are no guarantees for the
+    returned data in terms of ordering.
+
+     Depending on `Accept` header data can be returned either as JSON or as bytes serialized by SSZ.
```

Comment:
was wondering if we wanna state this here, seems a bit redundant considering we have a more general note about this now since https://github.com/ethereum/beacon-APIs/pull/457 and it's already implied by the response content type + the Accept header is mentioned there as well.

The reason to add the note is that it seems consistent with other endpoint that support SSZ, however could consider cleaning those up a bit

Reply:
i'd be happy to just merge and cleanup as a separate task...